src/tracing/server.coffee

```diff
@@ -0,0 +1,25 @@
+###
+This server response to ui on a /vizc end point and returns queried JSON as reponse
+###
+express = require 'express'
+bodyparser = require 'body-parser'
+varz = require 'express-varz'
+{conf} = require 'rainier/conf'
+port = conf.get 'vizc_server_port'
+vizJsonCreator = require './vizceral_json_file_creator_nodes'
+
+server = express()
+server.use bodyparser.json limit: '10mb'
+server.use bodyparser.urlencoded extended: true
+
+server.get '/vizc', (req, res, next) ->
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "X-Requested-With");
```

Comment:
The first one helps me in doing testing, will remove it. As I have deployed the server on vagrant and run the vizceral server on local laptop. Removing the second one, was unnecessary.